offerzen/global/employers/employers@9.0.js

```diff
@@ -0,0 +1,1689 @@
+(function () {
+  // FORM TARGETTING
+  const leadFormId = '#Original-Company-Lead-Form';
+  const pageRef = 'onSubmitOriginalCompanyLeadForm';
+  const formSubmitButtonSelector = 'input[type=submit]';
+  // END FORM TARGETING
+
+  // This needs to run regardless of whether JQuery has loaded
+  // Disable button
+  const button = document.querySelector(
+    `${leadFormId} ${formSubmitButtonSelector}`
+  );
+  button.setAttribute('disabled', 'disabled');
+  const label = button.value;
+  button.value = 'Loading...';
+
+  function enableSubmitButton() {
+    button.value = label;
+    button.removeAttribute('disabled');
+  }
+
+  // ------------------------------------------------------------------------------------------
+  window.$loaded(function () {
+    window.hasLoadedSelect2 = window.hasLoadedSelect2 ?? false;
+
+    window.$parsleyLoaded = function (cb) {
+      setTimeout(() => {
+        if (
+          window.Parsley &&
+          window.ParsleyValidator &&
+          window.hasLoadedSelect2
+        ) {
+          cb(window, document, parsley, undefined);
+          return;
+        }
+        $parsleyLoaded(cb);
+      }, 50);
+
+      if (!window.hasLoadedSelect2) {
+        var script = document.createElement('script');
+        script.onload = function onSelect2Loaded() {
+          if ($('body').select2) {
+            window.hasLoadedSelect2 = true;
+            return;
+          }
+          setTimeout(onSelect2Loaded, 50);
+        };
+        script.setAttribute(
+          'src',
+          'https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/js/select2.min.js'
+        );
+        script.setAttribute('async', true);
+        document.getElementsByTagName('head')[0].appendChild(script);
+      }
+    };
+
+    window.$parsleyLoaded(function (window, document, parsley) {
+      const form = $(leadFormId);
+
+      const formSubmitButton = form.find(formSubmitButtonSelector);
+
+      function getRoleTypes(formProperties) {
+        let fields = [
+          'role_type[In-office]',
+          'role_type[Fully-remote]',
+          'role_type[Hybrid]',
+        ];
+        let roleTypes = [];
+        form.find('.js-val-msg-roles').hide();
+
+        for (let key in formProperties) {
+          if (fields.includes(key) && formProperties[key] === 'on') {
+            if (key === 'role_type[Fully-remote]') {
+              roleTypes.push(
+                key.replace('role_type[', '').replace(']', '').replace('-', ' ')
+              );
+            } else {
+              roleTypes.push(key.replace('role_type[', '').replace(']', ''));
+            }
+            delete formProperties[key];
+          }
+        }
+        if (roleTypes.length < 1) {
+          form.find('.js-val-msg-roles').show();
+        }
+
+        return roleTypes;
+      }
+
+      function trackSubmission() {
+        var emailValue = form.find('#contact_email').val();
+        var isPlaywrightTest = emailValue.includes('playwrighttest');
```

Comment:
```suggestion
        var isPlaywrightTest = emailValue.match(/^\s*playwrighttest@offerzen\.com\s*$/i) != null;
```

Reply:
This will match things like `   playwrighttest@offerzen.com` but not `playwrighttest@blah.com`